packages/core/src/agent/index.ts

```diff
@@ -3184,6 +3184,8 @@ Message ${msg.threadId && msg.threadId !== threadObject.id ? 'from previous conv
                 ...result,
                 runId,
                 messages: messageList.get.response.aiV5.model(),
+                usage: payload.usage,
+                totalUsage: payload.totalUsage,
               } as any);
```

Comment:
while you're in there any chance you can remove `as any`? Not sure if that will have a cascade of fixes needed, but if not it'd be good to remove here